src/query/api/v1/handler/influxdb/write.go

```diff
@@ -0,0 +1,255 @@
+/*
+ * Author: Markus Stenberg <mstenber@aiven.io>
+ *
+ * Copyright (c) 2019 Aiven Oy
+ */
+
+package influxdb
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	imodels "github.com/influxdata/influxdb/models"
```

Comment:
nit: our linter enforces the following grouping format:

// stdlib imports

// m3db imports

// other imports (here that would be `imodels "github.com/influxdata/influxdb/models"` and `"go.uber.org/zap"`